arangod/Aql/QueryOptions.cpp

```diff
@@ -201,6 +202,9 @@ void QueryOptions::fromVelocyPack(VPackSlice slice) {
   if (value = slice.get("stream"); value.isBool()) {
     stream = value.getBool();
   }
+  if (value = slice.get("retriable"); value.isBool()) {
+    retriable = value.getBool();
```

Comment:
Can be done for this assignment and the ones above and below it:
```suggestion
    retriable = value.isTrue();
```
Difference: `getBool()` checks if the slice contains a boolean and will throw otherwise. Because we already established that the slice contains a boolean, we can directly call `isTrue()` on the slice.